### DISCUSSION TITLE
Ask: overwrite A record response within condition

### DISCUSSION THREAD

**Discussion by VDuyHoang** [TechnitiumSoftware/DnsServer#1077] [Q&A] [answered]
Hi,
Example: with ANY domain resolve to ip in range example: X.X.X.X/24 it will overwrite the response to static ip: B.B.B.B or CNAME: ip-filter...
Thanks

**Comment by ShreyasZare** [comment]
Thanks for the request. Something similar is possible by using the Address Translation feature in Split Horizon app. It would be great if you can explain your scenario a bit more in details so that I can understand it better.

**Reply by VDuyHoang** [reply]
Thanks for your response and your excellent software. I understand that the Split Horizon app will respond with different A and AAAA records based on the client network and with in a defined zone.
In my scenario, many ad domains are created daily with random names but they share the same subnet. Rather than collecting these domain names, I want any domain with an IP in that subnet to be detected and resolved to an IP I set.
if it could be done with Split Horizon app, please give me some instructions
Regards

**Reply by ShreyasZare** [reply, marked as answer]
Thanks for the compliments. The Split Horizon app also includes Address Translation feature which applies globally for all responses. It has a external to internal network/IP mapping so it will replace the IP address in outgoing responses. But right now, the feature allows configuring IP to IP and network to network mapping and thus network to IP mapping is not available. 

I guess, such a setup is also possible with iptables or similar tools that can be configured on your router. If that is feasible then you can try to explore it. Otherwise, the address translation feature can be updated to have this support.

**Reply by VDuyHoang** [reply]
Thank you, with a bit hack The Split Horizon app is working as i want 
Regards
